src/components/Header/User/UserMobile.tsx

```diff
@@ -98,13 +98,13 @@ const UserMenu = () => {
 export const UserMobile = () => {
   return (
     <div
-      className="fixed right-0 top-12 z-40 flex w-[260px] flex-col overflow-y-auto border-gray-300 bg-gray-100 dark:border-gray-900 dark:bg-gray-700 md:hidden"
+      className="fixed right-0 top-12 z-40 flex w-[260px] flex-col overflow-y-auto border-primary bg-layer-3 md:hidden"
       style={{ height: 'calc(100% - 48px)' }}
     >
       <UserInfo />
       <UserMenu />
       <div className="grow"></div>
-      <div className="border-t border-gray-300 p-4 dark:border-gray-900">
+      <div className="border-t border-primary p-4">
```

Comment:
it looks like `border-tertiary` should be used here and above
![image](https://github.com/epam/ai-dial-chat/assets/20163971/d9da7a13-c5e5-44ec-b99e-56db9a354f4f)